AutomatedTesting/Gem/PythonTests/prefab/TestSuite_Main.py

```diff
@@ -31,7 +35,68 @@ def test_PrefabLevel_OpensLevelWithEntities(self, request, workspace, editor, la
         from . import PrefabLevel_OpensLevelWithEntities as test_module
         self._run_prefab_test(request, workspace, editor, test_module)
 
-    def test_PrefabLevel_BasicWorkflow(self, request, workspace, editor, launcher_platform):       
-        from . import PrefabLevel_BasicWorkflow as test_module
-        self._run_prefab_test(request, workspace, editor, test_module)
 
+TEST_DIRECTORY = os.path.dirname(__file__)
+LOG_MONITOR_TIMEOUT = 180
+PREFAB_TEST_LEVEL_FOLDER = "Prefab"
+PREFAB_TEST_BASE_LEVEL_NAME = "Base"
+PREFAB_TEST_BASE_LEVEL_FILE_PATH = os.path.join(PREFAB_TEST_LEVEL_FOLDER, PREFAB_TEST_BASE_LEVEL_NAME)
+
+
+def test_Prefab_BasicWorkflow(request, editor, level, 
+                              test_file_name, expected_lines, timeout=LOG_MONITOR_TIMEOUT):
+    hydra.launch_and_validate_results(
+        request, TEST_DIRECTORY, editor, 
+        test_file_name, expected_lines, 
+        unexpected_lines=UnexpectedResults, cfg_args=[level], auto_test_mode=False, enable_prefab_system=True,
+        timeout=timeout,
+    )
```

Comment:
This is still using hydra.launch_and_validate_results(). Please don't use this as its deprecated, use self._run_prefab_test() instead